### DISCUSSION TITLE
Lithium 48V Battery Balancer / Equaliser  - 48V (2x 24V Batteries)

### DISCUSSION THREAD

**Discussion by Manos1966** [hoylabs/OpenDTU-OnBattery#529] [General]
Ich poste das Bild hier weil ich glaube, wir haben bis heute einen Balancer fuer 48V nicht in Deutschland gefunden.
Jemand hat danach gefragt, ich finde das Post aber nicht.
Das Bild ist aus Sued-Afrika
![Battery Balancer 2x24V](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/129660360/746fdf2d-a00b-40cd-ade5-f3bc4b5e73c4)
-Africa...

**Comment by swingstate** [comment]
Hab mal gefragt ob die nach Deutschland verschicken.

**Reply by Manos1966** [reply]
@swingstate warst du der jenige der danach gefragt hat? Brauchst du einen?

**Reply by swingstate** [reply]
ja

**Reply by Manos1966** [reply]
@swingstate ich habe jemanden in Duesseldorf kennengelernt, der naechste Woche zwei Stueck geliefert bekommt. 
Ich gehe davon aus du koenntest das zweite fuer ca 55EUR inkl Versand haben. 
Bei Interesse sag Bescheid.
